File: task-03/C/SubTask_4.c
#include <stdio.h>
#include <math.h>

int main() {
    FILE *input,*output;
    char ch;
    input = fopen("input.txt","r");
    output = fopen("output.txt","a");
    ch=fgetc(input);
    fclose(input); 
    double n=ch-'0';
    n=(int)ceil((double)n/2);
    for(int i=1;i<=n;i++) {
        for(int y=1;y<=n-i;y++) {
            fputc(' ',output);
        }
        for(int x=1;x<2*i;x++) {
            fputc('*',output);
        }
        fputc('\n',output);
    }
    for(int i=n-1;i>=1;i--) {
        for(int y=1;y<=n-i;y++) {
            fputc(' ',output);
        }
        for(int x=1;x<2*i;x++) {
            fputc('*',output);
        }
        fputc('\n',output);
    }
    fclose(output);
    return 0;
}

File: task-03/Javascript/SubTask_4.js
const fs=require('fs');
const input='input.txt';
const output='output.txt';
try {
    let n=fs.readFileSync(input,'utf8');
    n=parseInt(n, 10);
    n=Math.ceil(n/2);
    for(let i=1;i<=n;i++) {
        let line = '';
        for(let y=1;y<=n-i;y++) {
            line+=" ";
        }
        for(let x=1;x<2*i;x++) {
            line+="*";
        }
        fs.appendFileSync(output,line+'\n');
    }
    for(let i=n-1;i>=1;i--) {
        let line="";
        for(let y=1;y<=n-i;y++) {
            line+=" ";
        }
        for(let x=1;x<2*i;x++) {
            line+="*";
        }
        fs.appendFileSync(output,line+'\n');   
    }
} 
catch(err) {
    console.error(err.message);
}